timescaledb/how-to-guides/replication-and-ha/configure-replication.md

```diff
@@ -0,0 +1,391 @@
+Additionally, in cases where any data loss is fatal, asynchronous replication
+might not provide enough of a durability guarantee. The PostgreSQL
+[`synchronous_commit`][postgres-synchronous-commit-docs] feature has several
+options with varying consistency and performance tradeoffs.
+
+In the `postgresql.conf` file, set the `synchronous_commit` parameter to:
+
+*   `on`: This is the default value. The server does not return `success` until
+    the WAL transaction has been written to disk on the primary and any
+    replicas.
+*   `off`: The server returns `success` when the WAL transaction has been sent
+    to the operating system to write to the WAL on disk on the primary, but
+    does not wait for the operating system to actually write it. This can cause
+    a small amount of data loss if the server crashes when some data has not
+    been written, but it does not result in data corruption. Turning
+    `synchronous_commit` off is a well known PostgreSQL optimization for
+    workloads that can withstand some data loss in the event of a system crash.
+*   `local`: Enforces `on` behavior only on the primary server.
+*   `remote_write`: The database returns `success` to a client when the WAL
+    record has been sent to the operating system for writing to the WAL on the
+    replicas, but before confirmation that the record has actually been
+    persisted to disk. This is similar to asynchronous commit, except it waits
+    for the replicas as well as the primary. In practice, the extra wait time
+    incurred waiting for the replicas significantly decreases replication lag.
+*   `remote_apply`: Requires confirmation that the WAL records have been written
+    to the WAL and applied to the databases on all replicas. This provides the
+    strongest consistency of any of the `synchronous_commit` options. In this
+    mode, replicas always reflect the latest state of the primary, and
+    replication lag is nearly non-existent.
+
+<highlight type="important">
+If `synchronous_standby_names` is empty, the settings `on`, `remote_apply`,
+`remote_write` and `local` all provide the same synchronization level, and
+transaction commits wait for the local flush to disk.
+</highlight>
+
+This matrix shows the level of consistency each mode provides:
+
+|Mode|WAL Sent to OS (Primary)|WAL Persisted (Primary)|WAL Sent to OS (Primary & Replicas)|WAL Persisted (Primary & Replicas)|Transaction Applied (Primary & Replicas)|
+|-|-|-|-|-|-|
+|Off|✅|❌|❌|❌|❌|
+|Local|✅|✅|❌|❌|❌|
+|Remote Write|✅|✅|✅|❌|❌|
+|On|✅|✅|✅|✅|❌|
+|Remote Apply|✅|✅|✅|✅|✅|
+
+The `synchronous_standby_names` setting is a complementary setting to
+`synchronous_commit`. It lists the names of all replicas the primary database
+supports for synchronous replication, and configures how the primary database
+waits for them. The `synchronous_standby_names` setting supports these formats:
+
+*   `FIRST num_sync (replica_name_1, replica_name_2)`: This waits for
```

Comment:
I find the syntax not totally clear here. I would guess that `num_sync` is an integer from the description, but I wouldn't be certain. An example may help?